content/en/docs/contribute/style/style-guide.md

```diff
@@ -631,6 +631,34 @@ You can remove ... | You can easily remove ...
 These steps ... | These simple steps ...
 {{< /table >}}
 
+### EditorConfig
+For maintaing a constant coding style we would need you to maintain a  .editorconfig file 
```

Comment:
As this is the style guide itself, we **must** [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).

Also, I don't think the Kubernetes project is actually insisting that contributors maintain a `.editorconfig`. Maybe what we're saying is that the repository includes a `.editorconfig` and that this may help people to follow the style guide.